src/main/webapp/view/view.js

```diff
@@ -12,6 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import {intializeDiscussion} from '../view/discussion/discussion.js';
+
```

Comment:
Can the newlines between imports be removed? Or was that the linter

Reply:
That was the linter